huaweicloud/services/elb/resource_huaweicloud_elb_listener.go

```diff
@@ -335,10 +378,10 @@ func resourceListenerV3Update(ctx context.Context, d *schema.ResourceData, meta
 	}
 
 	// lintignore:R019
-	if d.HasChanges("name", "description", "ca_certificate", "default_pool_id",
-		"idle_timeout", "request_timeout", "response_timeout", "server_certificate",
-		"access_policy", "ip_group", "forward_eip", "tls_ciphers_policy",
-		"sni_certificate", "http2_enable", "advanced_forwarding_enabled") {
+	if d.HasChanges("name", "description", "ca_certificate", "default_pool_id", "idle_timeout",
```

Comment:
```suggestion
      xxxChanges := []string{
		"name",
		"description",
		xxx
	}
	if d.HasChanges(xxxChanges...)
```
remove  `// lintignore:R019`

Reply:
fixed